packages/rich-text/src/to-dom.js

```diff
@@ -110,6 +105,33 @@ function remove( node ) {
 	return node.parentNode.removeChild( node );
 }
 
+function createBogusBR( doc ) {
+	const br = doc.createElement( 'br' );
+	br.setAttribute( 'data-mce-bogus', '1' );
```

Comment:
I'm sure it's known / been discussed before, but this is quite the implied dependency to TinyMCE.

Reply:
Moved to `RichText`.

Reply:
Is the idea of a "line padding" something which is broadly applicable to support rich text editing? Still seems a bit oriented to TinyMCE specifically, even if not directly by name in the `rich-text` package.

Reply:
Is it something that could be contained within the `createEmpty` procedure?

Reply:
> Is the idea of a "line padding" something which is broadly applicable to support rich text editing?

Yes, it's not specific to TinyMCE. It should be part of the output to edit so you can always place the caret correctly in empty lines.